vista_2d/vista_2d_tutorial_monai.ipynb

```diff
@@ -0,0 +1,1405 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.    import cell_sam_wrapper
</pre><p>May I know why need this L22 and L23 here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Good catch, removed